docs/source/conf.py

```diff
@@ -144,13 +145,13 @@
 #
 # on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
 # http://docs.readthedocs.org/en/latest/faq.html?highlight=autodoc#how-do-i-change-behavior-for-read-the-docs
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-if on_rtd:
-    html_theme = 'default'
+if os.environ.get("READTHEDOCS"):
```

Comment:
What?!  We have been using an undefined variable `on_rtd` all this time?!  Did the pre-commit catch that or you did?